spec/acceptance/letsencrypt_plugin_dns_route53_spec.rb

```diff
@@ -1,16 +1,6 @@
 require 'spec_helper_acceptance'
 
-describe 'letsencrypt::plugin::dns_route53' do
-  supported = case fact('os.family')
-              when 'Debian'
-                # Debian 9 has it in backports, Ubuntu started shipping in Bionic
-                fact('os.release.major') != '9' && fact('os.release.major') != '16.04'
-              when 'RedHat'
-                true
-              else
-                false
-              end
-
+describe 'letsencrypt::plugin::dns_route53', if: %w[Debian RedHat].include?(fact('os.family')) do
```

Comment:
You can drop this too
```suggestion
describe 'letsencrypt::plugin::dns_route53' do
```